src/Simulation/qdk_sim_rs/README.md

```diff
@@ -148,3 +143,5 @@ TODO
 - Stabilizer simulation not yet exposed via C API.
 - Test and microbenchmark coverage still incomplete.
 - Too many APIs `panic!` or `unwrap`, and need replaced with `Result` returns instead.
+
+# Crate features
```

Comment:
Yes, the rest is filled in by the `document-features` crate based on documentation comments in `Cargo.toml` itself.

Reply:
I see, I assume it's appending it to the end of the file (rather than looking for a specific header name)? A comment in the readme could help avoid accidentally deleting the section or adding sections after it.

Reply:
That's right, yeah. There's two `#![cfg_attr(..., doc = ...)]` attributes at https://github.com/microsoft/qsharp-runtime/pull/920/files#diff-09741599828fb3d312b03bab5767da1f994d179496ea35a281db89a47339aebfR4, which causes rustdoc to append the contents of each attribute together. In particular, the first `#![doc(...)]` attribute uses `include_str` to bring in README.md, while the second one calls `document_features`.

Will add a note to that effect, then, as you suggest.